services/director-v2/src/simcore_service_director_v2/core/application.py

```diff
@@ -95,6 +102,14 @@ def create_base_app(settings: Optional[AppSettings] = None) -> FastAPI:
     logging.root.setLevel(settings.LOG_LEVEL.value)
     logger.debug(settings.json(indent=2))
 
+    # keep mostly quiet noisy loggers
+    quiet_level: int = max(
+        min(logging.root.level + LOG_LEVEL_STEP, logging.CRITICAL), logging.WARNING
+    )
+
+    for name in NOISY_LOGGERS:
+        logging.getLogger(name).setLevel(quiet_level)
+
```

Comment:
May I ask for this logic to me moved to the `packages/service-library/src/servicelib/rabbitmq_utils.py`
I'd like to reuse it in the dynamic-sidecar as well.

Reply:
https://github.com/ITISFoundation/osparc-simcore/blob/ccec5a6dae410413562f3ecc111598e130dc95ca/services/web/server/src/simcore_service_webserver/log.py
 for reference